src/test/java/com/microsoft/sqlserver/jdbc/fmtOnly/ParameterMetaDataTest.java

```diff
@@ -0,0 +1,151 @@
+package com.microsoft.sqlserver.jdbc.fmtOnly;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ParameterMetaData;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.microsoft.sqlserver.jdbc.TestResource;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import com.microsoft.sqlserver.testframework.Constants;
+
+
+public class ParameterMetaDataTest extends AbstractTest {
+
+    private static final String tableName = "[test_jdbc_" + UUID.randomUUID() + "]";
+
+    @BeforeEach
+    public void setupTests() throws SQLException {
+        try (Connection c = DriverManager.getConnection(AbstractTest.connectionString);

```

Comment:
Use the static `connection` instance from Abstract Test class for table creation/drop.